meilisearch-release-changelog/v1.3.0.md

```diff
@@ -0,0 +1,296 @@
+    "vectorStore": true
+  }'
+
+```
+
+### Sending Vectorized Documents
+
+For the first iteration of this feature you must compute the vectors on your side before sending to Meilisearch. Vectors should be sent with the document itself using the `_vectors` field. Multiple vectors can be sent for a single document.
+
+⚠️ Note the size of the vectors must be the same across all of your documents. If the sizes are inconsistent Meilisearch will show an error in your tasks.
+
+```bash
+curl -X POST -H 'content-type: application/json' \
+'localhost:7700/indexes/myvectors/documents' \
+--data-binary '[
+    { "id": 0, "_vectors": [0, 0.8, -0.2], "text": "Hello World" },
+    { "id": 1, "_vectors": [1, -0.2, 0], "text": "Hello Doggy" },
+    { "id": 2, "_vectors": [[0.5, 3, 1], [-0.2, 4, 6]], "text": "Hello People" }
+]'
+```
+
+### Query Meilisearch using Vectors
+
+You can query the documents with the nearest vector using the default `/search` route by sending the vector of your query via the `vector` field. Again, for the first iteration you must compute the vector query on your side before requesting from Meilisearch.
+
+```bash
+curl -X POST -H 'content-type: application/json' \
+'localhost:7700/indexes/myvectors/search' \
+--data-binary '{ "vector": [0, 1, 2] }'
+```
+
+### Similarity score
+As part of v1.3.0 Meilisearch will return a similarity score as part of the documents retrieved. This score will be between 0 and 1 and viewed under the `_semanticScore` property. The higher the ranking score, the more similar the document.
+
+```json
+{
+  "hits": [
+    { "id": 0, "_vectors": [0, 0.8, -0.2], "text": "Hello World", "_semanticScore": 0.6754 },
+    { "id": 1, "_vectors": [1, -0.2, 0], "text": "Hello Doggy", "_semanticScore": 0.7546 },
+    { "id": 2, "_vectors": [[0.5, 3, 1], [-0.2, 4, 6]], "text": "Hello People", "_semanticScore": 0.78 }
+  ],
+  "query": "",
+  "vector": [0, 1, 2],
+  "processingTimeMs": 0,
+  "limit": 20,
+  "offset": 0,
+  "estimatedTotalHits": 2
+}
+```
+
+
+🗣️ Since this feature is experimental, we need your help to iterate on it! You are more than welcome to share your thoughts and your feedback on [this discussion](https://github.com/meilisearch/product/discussions/677).
+
+⚠️ The experimental feature API can break between Meilisearch versions. Be careful if using it in production.
+
+Done by @Kerollmops in #3825
+
+
+## Display ranking scores at search
+
+Meilisearch v1.3.0 introduces ranking scores per document via the `showRankingScore` search query parameter. 
+
+```bash 
+curl \
+  -X POST 'http://localhost:7700/indexes/movies/search' \
+  -H 'Content-Type: application/json' \
+  --data-binary '{ "q": "Batman Returns", "showRankingScore": true }'
+```
+
+Using this search query parameter means that each document will return with a `_rankingScore` property displaying a score between 0 and 1. The higher the ranking score, the more relevant the document.
+
+```json 
+"_rankingScore": 0.8575757575757575,
+```
```

Comment:
Not sure if this part is really usefull thanks to the sentence right above

Reply:
it does give an example is the only thing, see discussion here: https://github.com/meilisearch/engine-team/pull/52#discussion_r1245124720 happy to remove if you think its not neccesary

Reply:
No I will not go against this, it's a tiny detail 👌